crates/server/src/services/core.rs

```diff
@@ -203,9 +203,9 @@ impl CoreService {
         while let Some(res) = initial.next().await {
             let InitialLeaf { leaf, checkpoint } = res?;
             data.log.push(&leaf);
-
+            let new_key = leaf.log_id.0.clone().try_into().unwrap();
```

Comment:
This doesn't appear to be any different from the existing `leaf.log_id.clone()`, or am I mistaken?

It appears to just be invoking `From<AnyHash> for LogId`, which we're cloning; cloning the `LogId` is just clearer to me.